gremlin-groovy/pom.xml

```diff
@@ -93,6 +93,10 @@ limitations under the License.
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Hm, It was needed for build.....
Now I can not reproduce the failure, so rollback it